doc/index.rst

```diff
@@ -0,0 +1,19 @@
+Welcome to yaml2sbml's documentation!
+===============================================
+
+
+This small package allows the user to [TODO]
```

Comment:
I will fix this

Reply:
I took the liberty of adding some text, but feel free to change it all (i just think it's easier to change something than to write from scratch) :)